__test__/api/comment-vote.test.js

```diff
@@ -27,19 +27,12 @@ describeIfEndpoint(
           ownerId: commentObject.ownerId,
         });
       expect(res.status).toBe(200);
-      expect(res.body.data.commentId).toEqual(String(comment._id));
```

Comment:
same here stash any changes you have and update forked/local repo, then reapply your work.
afterwards you can pop your stash